esrun_data/crop/mozambique/dataset.json

```diff
@@ -0,0 +1,184 @@
+          "format": {
+            "name": "geotiff"
+          }
+        }
+      ],
+      "type": "raster"
+    },
+    "prediction_v1": {
+      "band_sets": [
+        {
+          "bands": [
+            "output"
+          ],
+          "dtype": "uint8"
+        }
+      ],
+      "type": "raster"
+    },
+    "sentinel1_ascending": {
+      "band_sets": [
+        {
+          "bands": [
+            "vv",
+            "vh"
+          ],
+          "dtype": "float32"
+        }
+      ],
+      "data_source": {
+        "cache_dir": "cache/planetary_computer",
+        "duration": "366d",
+        "ingest": false,
+        "name": "rslp.satlas.data_sources.MonthlySentinel1",
+        "query": {
+          "sar:instrument_mode": {
+            "eq": "IW"
+          },
+          "sar:polarizations": {
+            "eq": [
+              "VV",
+              "VH"
+            ]
+          },
+          "sat:orbit_state": {
+            "eq": "ascending"
+          }
+        },
+        "query_config": {
+          "max_matches": 12
+        },
+        "time_offset": "-180d"
+      },
+      "type": "raster"
+    },
+    "sentinel1_descending": {
+      "band_sets": [
+        {
+          "bands": [
+            "vv",
+            "vh"
+          ],
+          "dtype": "float32"
+        }
+      ],
+      "data_source": {
+        "cache_dir": "cache/planetary_computer",
+        "duration": "366d",
+        "ingest": false,
+        "name": "rslp.satlas.data_sources.MonthlySentinel1",
+        "query": {
+          "sar:instrument_mode": {
+            "eq": "IW"
+          },
+          "sar:polarizations": {
+            "eq": [
+              "VV",
+              "VH"
+            ]
+          },
+          "sat:orbit_state": {
+            "eq": "descending"
+          }
+        },
+        "query_config": {
+          "max_matches": 12
+        },
+        "time_offset": "-180d"
+      },
+      "type": "raster"
+    },
+    "sentinel2": {
+      "band_sets": [
+        {
+          "bands": [
+            "B02",
+            "B03",
+            "B04",
+            "B08"
+          ],
+          "dtype": "uint16"
+        },
+        {
+          "bands": [
+            "B05",
+            "B06",
+            "B07",
+            "B8A",
+            "B11",
+            "B12"
+          ],
+          "dtype": "uint16",
+          "zoom_offset": -1
+        },
+        {
+          "bands": [
+            "B01",
+            "B09"
+          ],
+          "dtype": "uint16",
+          "zoom_offset": -2
+        }
+      ],
+      "data_source": {
+        "cache_dir": "cache/planetary_computer",
+        "duration": "366d",
+        "harmonize": true,
+        "ingest": false,
+        "max_cloud_cover": 50,
+        "name": "rslp.satlas.data_sources.MonthlyAzureSentinel2",
+        "query_config": {
+          "max_matches": 12
+        },
+        "sort_by": "eo:cloud_cover",
+        "time_offset": "-180d"
```

Comment:
Growing season is October - July. This is the date range for which the labels are relevant.